test/unit/CMakeLists.txt

```diff
@@ -43,13 +43,14 @@ DLAF_addTest(test_matrix
   MPIRANKS 6
 )
 
-# Generic libraries
+#Generic libraries
```

Comment:
I think you have a problem in your way of checking file styles...please, give it a check.